addon/components/map-commands-dialogs/search-settings/kml.js

```diff
@@ -0,0 +1,98 @@
+/**
+  @module ember-flexberry-gis
+*/
+
+import Ember from 'ember';
+import layout from '../../../templates/components/map-commands-dialogs/search-settings/kml';
+
+/**
+  Search settings part of KML layer modal dialog.
+
+  @class KMLSearchSettingsComponent
+  @extends <a href="http://emberjs.com/api/classes/Ember.Component.html">Ember.Component</a>
+*/
+export default Ember.Component.extend({
+
+  _searchPropertiesArray: Ember.computed('searchProperties', function () {
```

Comment:
Useless. Either remove it or add dropdown with attributes list into template

Reply:
fixed